manage.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+from service import app, db
+
+# db.create_all() needs all models to be imported explicitly (not *)
+from service.models import TitleNumbersUprns, TitleRegisterData
```

Comment:
TitleNumbersUprns should die as a table/model: we have an upcoming chore to clean that up though :) let's just not do additional migration work for that one maybe?